src/main/java/com/microsoft/sqlserver/jdbc/Geometry.java

```diff
@@ -22,17 +27,20 @@
      *         if an exception occurs
      */
     private Geometry(String WellKnownText, int srid) throws SQLServerException {
```

Comment:
First character in param name must not be capitalized. Please change to 'wellKnownText'